py/terra_widgets/html_snapshots.py

```diff
@@ -342,6 +331,17 @@ def on_choose_workspace(changed):
 def display_html_snapshots_widget():
   """Create an ipywidget UI encapsulating all three UIs related to HTML snapshots."""
 
+  # Configure notebook display preferences to better suit this UI.
+  if pd.__version__.startswith('1'):
+    pd.set_option('display.max_colwidth', None)
+  else:
+    pd.set_option('display.max_colwidth', -1)
+  get_ipython().run_cell_magic(
```

Comment:
Could you add a comment to specify the scope of this change? i.e. does this affect all cells in the notebook? Presumably the change is at least scoped to the active kernel.

I'm guessing there is no way to change this for just the current output cell, which would be ideal.

Reply:
This is the straightforward way to accomplish the desired display changes programmatically. I'll keep an eye out for other approaches that are specific to a cell.